catalogue/tests/conftest.py

```diff
@@ -0,0 +1,23 @@
+import pytest
+
+
+def pytest_addoption(parser):
+    parser.addoption("--slow", action="store_true", help="include slow tests")
+
+
+@pytest.fixture()
+def pathy_fixture():
```

Comment:
We'll need to install `pathy` on the CI to ensure that the related test is actually ever executed